### PR TITLE
added the notification core component

### PR DESCRIPTION
**Pull request by SakaRicky**
- on create, delete, update weight, shows notification for 5s

### CODE REVIEW

File: src/components/Core/Notifications/index.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    const inlineStyles = success ? {
+        backgroundColor: "#e0ffe3",
+        color: "#00780c"
+    } : {
+        backgroundColor: "#ffd9d9",
+        color: "#9c0000"
+    };
+    
+    return (
+        <div style={inlineStyles} className={styles.notification}>
```
Comment (rolandgeider) [inline src/components/Core/Notifications/index.tsx:20]: Perhaps we can just use an alert from MUI? Then we don't have to add any own CSS classes? Or perhaps a snackbar? Snackbars might be too "far" from the form however

https://mui.com/components/alert/
https://mui.com/components/snackbars/

File: src/types.ts [hunk 2]
```diff
@@ -2,4 +2,10 @@ export interface BodyWeightType {
     id: number,
     date: string,
     weight: string,
+}
+
+export interface Notification {
+    notify: boolean
+    message: string
+    success: boolean
```
Comment (rolandgeider) [inline src/types.ts:10]: If we use the alert component, we could pass the type here (error, warning, info, success)

File: src/components/Core/Notifications/index.tsx (hunk 1 above)
Comment (SakaRicky) [reply src/components/Core/Notifications/index.tsx:20]: I feel alert is better

File: src/types.ts (hunk 2 above)
Comment (SakaRicky) [reply src/types.ts:10]: Ok 👍🏽

File: src/components/BodyWeight/Form/WeightForm.tsx [hunk 3]
```diff
@@ -26,11 +26,33 @@ export const WeightForm = ({ weightEntry, closeFn }: WeightFormProps) => {
     const updateWeightEntry = useCallback(async (entry: WeightEntry) => {
         const action = { type: SetState.UPDATE_WEIGHT, payload: entry };
         dispatch(action);
+        dispatch(setNotification(
+            {
+                notify: true,
+                message: "Weight updated Successfully",
```
[the remark this answers is not in the document]
Comment (SakaRicky) [reply src/components/BodyWeight/Form/WeightForm.tsx:32]: Yeah that's true. Didn't thought of it

**Pull request merged**